addons/l10n_il/data/account_account_tag.xml

```diff
@@ -19,4 +19,8 @@
     <record id="account_tag_retention_tax_customers_account" model="account.account.tag">
         <field name="name">Withholding Customers Tax Account</field>
     </record>
+
+    <record id="account_tag_vat_input_import_line" model="account.account.tag">
```

Comment:
No way to use the VAT closing accounts on the tax group instead?  (or on the CoA)

Reply:
@jco-odoo 
reverted this tag change for now, until we have a better solution.